kivy/uix/modalview.py

```diff
@@ -102,25 +105,16 @@ class ModalView(AnchorLayout):
     .. versionchanged:: 2.0.0
         Added property 'overlay_color'.
 
-    '''
+    """
 
-    auto_dismiss = BooleanProperty(True)
+    auto_dismiss = BooleanProperty()
     '''This property determines if the view is automatically
     dismissed when the user clicks outside it.
 
     :attr:`auto_dismiss` is a :class:`~kivy.properties.BooleanProperty` and
     defaults to True.
     '''
 
-    attach_to = ObjectProperty(None)
```

Comment:
it might not be an api break, because if anyone sets a value for this, the parent window will be the main one anyway, but do we need to document that the property now does nothing? It's unlikely that people bound anything to this property, but it would break their code if they did, so maybe leaving it in with a statement that it has no behavior now and will be removed in the future would be better?

Reply:
Thank you very much @tshirtman  for your review/approval!!! :+1: 

Getting your concerns and therefore just added (in the 4rd/last commit) a `versionchanged` to
the docstring of the modalview module regarding the removal of `attach_to` (instead of marking
it as deprecated, to keep the Kivy code base more compact and clean).

As you said setting/using this property will make no difference, so an old project using it could
easily be fixed by just removing any reference to `ModalView.attach_to` - hope you agree!!

Reply:
Yeah, I am similarly concerned by the API break.

And like @tshirtman suggested, it's better to leave the property in place saying it's not used anymore and remove it in a latter version. rather than abruptly dropping it. 

It can be done by adding `deprecated` arg as follows: `attach_to = ObjectProperty(None, deprecated=True)`. And update the docs that it is not yet dropped, but will be in the future.